src/components/Radio/StyledRadio.js

```diff
@@ -8,7 +8,7 @@ const Label = styled.label`
 `
 
 const Field = styled.input`
-  margin-right: 5px;
+  margin-right: 10px;
```

Comment:
I think this change is fine. On RAD we gave added an explicit value of zero to the other margins to reset any browser-set values. I'm wondering if we should do that for this as well? Or would it be better to set that in the app?

Reply:
Initially I didn't want to give it any margin either but Nakul specifically asked for it to match the wireframes so I basically doubled it here.

Reply:
> Initially I didn't want to give it any margin either but Nakul specifically asked for it to match the wireframes so I basically doubled it here.

OK, no problem, though there seems to have been a little confusion throughout this project on whether the wireframes or existing RAD/MAS styles are being followed.